app/views/home/index.html.erb

```diff
@@ -1 +1,40 @@
-<h1 id="welcome">Making HPC Greener</h1>
+<h1 id="welcome">Making HPC greener</h1>
+<p id="welcome-tagline">
+  Finding the most carbon efficient HPC solutions by ranking the emissions of real servers.
+</p>
+
+<%= button_to "Enter your server data", root_path, class: 'button big-text my-4', method: :get %>
+
+<div class="home-top-section my-4">
+  <%= render 'counter', locals: { number: @num_devices, text: 'servers entered' } %>
+
+  <div class="cl-card flex-column-centred">
+    <h3>Top most carbon efficient servers</h3>
+    <p>Ranked by estimated equivalent CO<sub>2</sub> emissions per core at full load.</p>
+    <% if @devices.empty? %>
+      <p class="big-text mt-5">No server data has been entered yet</p>
+    <% else %>
+      <div class="home-leaderboard leaderboard-wrapper mt-4">
+        <div class="leaderboard-content-wrapper">
+          <% @devices.each_with_index do |device, index| %>
+            <div class="leaderboard-item-wrapper blurred-frame rank-<%= index + 1 %>">
```

Comment:
Could I ask that clicking on these device entries links you to the relevant device's own page? This is accessible at `/device/<%= device[:display_name] %>` but I don't really know enough about the frontend implementation to make it link properly :sweat_smile:

Reply:
Updated to add the links on the home page and full leaderboards :+1: